app/assets/src/components/views/SampleViewV2/ReportTableV2.jsx

```diff
@@ -0,0 +1,404 @@
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 60,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [(rowData.nt || {}).rpm || 0, (rowData.nr || {}).rpm || 0];
+        },
+        cellRenderer: ({ cellData }) =>
+          this.renderNtNrDecimalValues({ cellData, decimalPlaces: 1 }),
+        dataKey: "rpm",
+        label: "rPM",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["rpm"],
+            nullValue: 0,
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 75,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [(rowData.nt || {}).count || 0, (rowData.nr || {}).count || 0];
+        },
+        cellRenderer: this.renderNtNrDecimalValues,
+        dataKey: "r",
+        label: "r",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["count"],
+            nullValue: 0,
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 75,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [
+            (rowData.nt || {}).contigs || 0,
+            (rowData.nr || {}).contigs || 0,
+          ];
+        },
+        cellRenderer: this.renderNtNrDecimalValues,
+        dataKey: "contigs",
+        label: "contig",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["contigs"],
+            nullValue: 0,
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 75,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [
+            (rowData.nt || {}).contig_reads || 0,
+            (rowData.nr || {}).contig_reads || 0,
+          ];
+        },
+        cellRenderer: this.renderNtNrDecimalValues,
+        dataKey: "contig_reads",
+        label: "contig r",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["contig_reads"],
+            nullValue: 0,
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 75,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [
+            (rowData.nt || {}).percent_identity || 0,
+            (rowData.nr || {}).percent_identity || 0,
+          ];
+        },
+        cellRenderer: ({ cellData }) =>
+          this.renderNtNrDecimalValues({ cellData, decimalPlaces: 1 }),
+        dataKey: "percent_identity",
+        label: "%id",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["percent_identity"],
+            nullValue: 0,
```

Comment:
I'm not sure about this `nullValue` implementation. This means that if you sort `ascending`, then all the rows with null value would be the top, since null is interpreted at 0. Is that what you want, or do you always want the rows with null value to be at the bottom of the table?

Reply:
Thats what i want. You should always be able to find the values even if null. So if you pushed them to the bottom when sorting ascending, they should be on top once you invert.
In some cases (with negative values) we might want to put them in the middle.